modules/gatsby/src/gatsby-node/process-nodes/process-nodes-markdown.js

```diff
@@ -49,6 +49,7 @@ module.exports.processNewMarkdownNode = function processNewMarkdownNode(
 
   // Update path
   let relPath = node.fields.slug;
+  let entry = relPath;
```

Comment:
Does entry ever change? Should it be `const`? Does our test setup not run linter?